compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -1076,18 +1078,9 @@ pub trait PrettyPrinter<'tcx>:
         // Builtin bounds.
         // FIXME(eddyb) avoid printing twice (needed to ensure
         // that the auto traits are sorted *and* printed via cx).
-        let mut auto_traits: Vec<_> =
+        let mut auto_traits: FxIndexSet<_> =
```

Comment:
This should keep being a `Vec`. Below you can sort by just the def path str using `sort_by_key()`.